README.md

```diff
@@ -17,21 +17,21 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] String
-[] Boolean
-[] Undefined
-[] NaN
-[] Number
-[] Array
-[] Null
+[x] String
+[x] Boolean
+[x] Undefined
+[x] NaN
+[x] Number
+[x] Array
+[x] Null
 ```
```

Comment:
`NaN` is not a data type. It's actually a type of number, which sounds counter-intuitive.